README.rst

```diff
@@ -15,6 +15,12 @@ You can try NeuroR's functionality on MyBinder:
 - Cut-plane detection - |cutplane|
 - Morphology repair - |repair|
 
+
+Acknowledgements
+----------------
+
+This research was supported by the EBRAINS research infrastructure, funded from the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant Agreement No. 945539 (Human Brain Project SGA3).
```

Comment:
```suggestion
This research was supported by the EBRAINS research infrastructure, funded by the European Union’s Horizon 2020 Framework Programme for Research and Innovation under the Specific Grant Agreement No. 945539 (Human Brain Project SGA3).
```

Reply:
again that is the official statement from HBP

Reply:
ok then